example/pubspec.lock

```diff
@@ -109,13 +109,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.6.4"
-  pedantic:
```

Comment:
Why did you remove `pedantic` package?

Reply:
I didn't... see the pubspec.yaml... This file is automatically generated. Let me do `flutter pub get` again.

Reply:
See, it is below. So it wasn't removed after all. Just reordered.

Reply:
If I open the file I can't find any pedantic word in it... Am I missing something?

Reply:
https://github.com/Alpha-health/horizontal_blocked_scroll_physics/blob/master/pubspec.lock#L98

Reply:
Oh, this is the example project's pubspec.lock! I'm not sure why pedantic got removed. My bad! Sorry for the confusion! Anyway, I don't think it matters that much since this file is completely managed by Flutter and it's only about the example. It has no impact in the library at all.

Reply:
I mean, I didn't even changed the `example/pubspec.yaml` at all so it probably was a dependency from another package.

Reply:
👍